Makefile

```
@@ -136,10 +149,10 @@ down: ## stop and remove backend containers
 	@$(COMPOSE) down
 .PHONY: down
 
-es-index: ## create elasticsearch index and sample documents
+es-index: ## create elasticsearch index
 es-index: run-es
 	@echo "Creating $(ES_INDEX) index"
-	bin/es index $(ES_INDEX)
+	curl -X PUT $(ES_URL)/$(ES_INDEX)
```

```suggestion
	curl -X PUT $(ES_URL)/$(ES_INDEX)
	curl -X PUT ${ES_URL}/$(ES_INDEX)/_settings -H 'Content-Type: application/json' -d '{"index": {"number_of_replicas": 0}}'
```
I really like your solution)
Could we also set the number of replicas to 0? It seems that ES refuses to exit the "yellow" state when it's unable to meet the default replication factor.

---

Good point!